### PR TITLE
refactor: show blockexplorer link on ln invoice page

### LINKED ISSUE #113: Reverse swap: show blockexplorer link on ln invoice page
image

blockexplorer link should be visible here with "success" message, since boltz automatically jumps to this page and then I don't have a chance to convince myself that the tx was really locked in:

image

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
closes #113

### CODE REVIEW

File: src/views/reverse/steps/payInvoice.js [hunk 1]
```diff
@@ -46,26 +46,34 @@ const payInvoiceStyles = () => ({
...
+    const { classes, swapInfo, swapResponse } = this.props;
+    const link = swapResponse
+      ? `${getExplorer(swapInfo.quote)}/${swapResponse.lockupTransactionHash}`
+      : '';
 
     return (
       <View className={classes.wrapper}>
         <View className={classes.qrcode}>
-          <QrCode size={300} link={invoice} />
+          <a href={link} target="_blank" rel="noopener noreferrer">
```
Comment (michael1011) [inline src/views/reverse/steps/payInvoice.js]: Please rebase the branch and use the `Link` component you added with [this commit](https://github.com/BoltzExchange/boltz-frontend/commit/efcaa53ac0def6c2cd3e3c806008a0bb5313cdbf)

File: src/views/reverse/steps/lockingFunds.js [hunk 2]
```diff
@@ -31,14 +26,12 @@ const LockingFunds = ({ classes, swapInfo, swapResponse }) => {
         to receive, this is important to keep the <br />
         swap atomic and trustless. It might take <br />
         up to 10 minutes. Please be patient. <br />
-        <a href={link}>Click here</a> to see the lockup transaction.
```
Comment (michael1011) [inline src/views/reverse/steps/lockingFunds.js]: Although I agree with #113 I do think that we should keep the block explorer link on this page. For the sake of verifying that the transaction was sent and not waiting for a transaction that doesn't exist to be confirmed.

Comment (ImmanuelSegol) [reply src/views/reverse/steps/lockingFunds.js]: @michael1011 So we should add the link on both pages?

Comment (michael1011) [reply src/views/reverse/steps/lockingFunds.js]: Yes, I would say so.

**Comment by peartobear** [comment]
I'd say moving the QR code little bit on top and putting the link down would be a better choice.

**Pull request merged**